app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,11 @@
+<div id="<%= dom_id category %>">
+  <div class="card m-2">
+    <div class="card-body">
+      <div class="d-flex">
+        <div class="px-4">
+          <%= link_to category.name, category %>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
## [OPTIONAL]

![image](https://user-images.githubusercontent.com/65068771/170394546-2aec279e-22d6-4959-96ce-e47432297a6f.png)

- [ ] Kindly I would really suggest you follow the original template design as this is one of the requirements. Make sure :
- The icon of the category is visible.
- Your nav-bar is well designed
- You use the same colors and design for the `new category button` :point_down: 
![image](https://user-images.githubusercontent.com/65068771/170394798-4cc25bda-976a-4ea7-ba86-c902467ce9cb.png)
- use a good icon for going back instead of `<` :point_down: 
![image](https://user-images.githubusercontent.com/65068771/170395346-baa0d0bd-3f52-4cdf-bd09-483c25d8ee79.png)